src/actions/index.js

```diff
@@ -0,0 +1,8 @@
+import weeatApi from '../api/weeatApi';
+import { FETCH_RESTAURANTS } from './types';
+
+export const fetchRestaurants = () => async dispatch => {
+    const response = await weeatApi.get('/restaurants');
+
+    dispatch({ type: FETCH_RESTAURANTS, payload: response.data });
```

Comment:
Worth knowing that for any API request, as it might fail or success, you'll want to dispatch at least three different kinds of actions: REQUEST / FAILURE / SUCCESS. You can read more about it [here](https://redux.js.org/advanced/async-actions).